src/Pages/Main/Component/AccomodationType/AccomodationType.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import { API } from '../../../../config';
+import styled from 'styled-components';
+
+const AccomodationType = ({ accomoType, setAccomoType }) => {
+  const [categoryType, setCategoryType] = useState([]);
+  const [currentId, setCurrentId] = useState(0);
+
+  const handleColor = (e, name) => {
+    setCurrentId(e.target.id);
+    setAccomoType(name);
+  };
+
+  useEffect(() => {
+    fetch(`${API}/hotels/main`)
+      .then(res => res.json())
+      .then(res => {
+        setCategoryType(res.results.category);
+      });
+  }, []);
+
+  return (
+    <>
+      <Wrapper>
+        {categoryType.map((img, index) => (
+          <ImageBox>
+            <Image
+              id={index}
+              src={img.img_url}
+              alt={img.name}
+              onClick={e => handleColor(e, `${img.name}`)}
+            />
+            <Title color={setCurrentId === index && 'color'}>{img.name}</Title>
```

Comment:
```suggestion
            <Title color={setCurrentId === index}>{img.name}</Title>
```
true / false 값을 사용한다면 이렇게 사용하는 것이 용법에 더 맞아보입니다

Reply:
아 제가 그동안 잘못생각하고 있었네요 감사합니다 수정하겠습니다!